tools/dtrace/get-ds-state.d

```diff
@@ -3,37 +3,40 @@
  * Exit after 5 seconds.
  */
 #pragma D option quiet
-#pragma D option strsize=1k
+#pragma D option strsize=2k
 
 /*
  * Translate the longer state string into a shorter version
  */
 inline string short_state[string ss] =
-    ss == "active" ? "ACT" :
-    ss == "new" ? "NEW" :
-    ss == "replaced" ? "RPL" :
-    ss == "live_repair_ready" ? "LRR" :
-    ss == "live_repair" ? "LR" :
-    ss == "faulted" ? "FLT" :
-    ss == "offline" ? "OFL" :
-    ss == "reconcile" ? "REC" :
-    ss == "wait_quorum" ? "WQ" :
-    ss == "wait_active" ? "WA" :
-    ss == "connecting" ? "CON" :
+    ss == "Active" ? "ACT" :
```

Comment:
Is there a way to share this function with all the other scripts?

Reply:
Maybe... Let me see if I can find an easy way.
I think it's "possible", but I don't want the callers of the scripts have to do too much, or remember too much.

Reply:
You could look at what [OPTE does](https://github.com/oxidecomputer/opte/tree/master/dtrace), which has a few shared type and function definitions for these kinds of things.